dsp-role/core/src/test/scala/dsp/role/sharedtestdata/RoleTestData.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © 2021 - 2022 Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package dsp.role.domain
+
+import dsp.valueobjects.Id
+import dsp.valueobjects.Permission
+import dsp.valueobjects.Role._
+
+/**
+ * Contains shared role test data.
+ */
+object RoleTestData {
+  val id1          = Id.RoleId.make()
+  val name1        = LangString.make("Name", "en")
+  val description1 = LangString.make("Description", "en")
+  val users1       = List(RoleUser(Id.UserId.make().fold(e => throw e.head, v => v)))
```

Comment:
True, but I didn't want to create value object for it simply because I don't know how it's going to looks like - if this is going to be complete `User` entity or something else. This should be delegate to finisher task, once we define the `Role` completely.